client/src/main/kotlin/io/spine/examples/pingh/client/LoginFlow.kt

```diff
@@ -107,185 +109,200 @@ public class LoginFlow internal constructor(
         }
         this.stage.value = stage
     }
-}
 
-/**
- * Represents a stage in the GitHub login process.
- */
-public interface LoginStage
-
-/**
- * Type of the stage in the GitHub login process.
- */
-private typealias LoginStageType = KClass<out LoginStage>
-
-/**
- * A stage of the login flow on which the user enters their GitHub username
- * and receives a user code in return.
- *
- * @property client Enables interaction with the Pingh server.
- * @property session The information about the current user session.
- * @property moveToNextStage Updates value of the current login stage.
- */
-public class EnterUsername internal constructor(
-    private val client: DesktopClient,
-    private val session: MutableStateFlow<UserSession?>,
-    private val moveToNextStage: (LoginStage) -> Unit
-) : LoginStage {
+    /**
+     * Represents a stage in the GitHub login process.
+     */
+    public interface LoginStage
```

Comment:
Has anything changed, so that we now have `LoginFlow.LoginStage` public interface?